Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptions.swift

```diff
@@ -14,6 +14,14 @@ extension ChatMessageLayoutOptions: Identifiable {
         // Since it is a Set, we need to sort it to make sure the value doesn't change per call.
         map(\.rawValue).sorted().joined(separator: "-")
     }
+    
+    public mutating func remove(_ options: ChatMessageLayoutOptions) {
+        self = subtracting(options)
+    }
+
+    public mutating func insert(_ options: ChatMessageLayoutOptions) {
+        options.forEach { self.insert($0) }
+    }
```

Comment:
nit: i would move those outside `ChatMessageLayoutOptions: Identifiable` extension as it's not related

Reply:
@evsaev done! :)